scripts/opencollab-scsv-meta

```diff
@@ -49,29 +108,34 @@ def decode(x):
 def escape(x):
     return x.encode("unicode-escape")
 
-operations = list()
-
-
-def callback(option, opt, value, parser, operation):
-    operations.append((operation, value))
 
 usage = "usage: %prog [options]"
 parser = OptionParser(usage=usage)
 
-parser.add_option("-G", "--get", action="callback",
-                  callback=callback, callback_args=("get",), type="string",
+parser.add_option("-G", "--get", type="string",
                   help=("Get metadata from Wiki with a  MetaTable argument " +
                         "string."))
-parser.add_option("-A", "--add", action="callback",
-                  callback=callback, callback_args=("add",), type="string",
+parser.add_option("-A", "--add", type="string",
                   help=("Add metadata based on a semicolon-delimited CSV file " +
                         "in a batch fashion. First row states the keys, " +
                         "first colums states the pages."))
-parser.add_option("-S", "--set", action="callback",
-                  callback=callback, callback_args=("set",), type="string",
+parser.add_option("-S", "--set", type="string",
                   help=("Replace metadata based on a semicolon-delimited CSV " +
                         "file in a batch fashion. First row states the keys, " +
                         "first colums states the pages."))
+parser.add_option("--multivalues",
+                  type="choice",
+                  choices=["duplicate-keys", "aggregate-values", "multi-row"],
+                  default="duplicate-keys",
+                  help=("How to handle keys with multiple values:\n" +
+                        "duplicate-keys: default, keys with multiple values get " +
+                        "duplicate keys in the header for each instance of value.\n" +
+                        "aggregate-keys: combine multivalues into a list for each row"
+                        "multi-row: split multivalued keys into multiple rows, "
+                        "by writing subsequent multivalues on each row until all"
+                        "the values have been output"))
```

Comment:
These lines yield the following help text:

```
  --multivalues=MULTIVALUES
                        How to handle keys with multiple values: duplicate-
                        keys: default, keys with multiple values get duplicate
                        keys in the header for each instance of value.
                        aggregate-keys: combine multivalues into a list for
                        each rowmulti-row: split multivalued keys into
                        multiple rows, by writing subsequent multivalues on
                        each row until allthe values have been output
```

Note the missing newlines and spaces in places like `each rowmulti-row: split multivalued`.